README.md

```diff
@@ -300,13 +300,13 @@ console.log(c.driverOptions)
 | Property | Type | Description | Default |
 | ----- | ---- | ---------- |  ----  |
 | `ssl` | Number/String | A boolean to enable or disables TLS/SSL for the connection | `undefined` |
-| `sslType` | String | The desired ssl strategy. Possible values: `NONE`, `SYSTEMCA`, `IFAVAILABLE`, `UNVALIDATED`, `SERVER`, `ALL` | `NONE` |
+| `sslMethod` | String | The desired ssl method. Possible values: `NONE`, `SYSTEMCA`, `IFAVAILABLE`, `UNVALIDATED`, `SERVER`, `ALL` | `NONE` |
```

Comment:
is this what the Node Driver also calling this property? If yes, then go for it for consistency!

otherwise I am not sure if these are technically `sslMethod`s? (correct me if I am wrong pls!) I thought ssl methods have to do with how you're going to do the handshake and the type of encryption. The possible values you have listed here make me think these are more like `sslOption`.

Reply:
oh i just read your description. ignore me! lol